package.json

```diff
@@ -9,32 +9,32 @@
     "url": "https://github.com/juju/juju-gui.git"
   },
   "dependencies": {
-    "classnames": "2.2.5",
-    "clipboard": "1.7.1",
+    "classnames": "^2.2.6",
+    "clipboard": "^1.7.1",
```

Comment:
This is not the latest version of clipboard. The latest version 2.x does not actually function because it tries to access a file which doesn't exist in its package. 

re: https://github.com/zenorocha/clipboard.js/issues/558